sale_configurator_option/models/account_move.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from odoo import api, models
+
+from odoo.addons.sale_configurator_base.models.sale import update_attrs
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move"
+
+    @api.model
```

Comment:
Why do we need a dynamic view here? Why not using pure xml?

Reply:
I think this is related to the ir.ui.view part and things like _get_sale_line_tree_item, _get_sale_line_form_item, which I don't fully understand